flink-runtime/src/main/java/org/apache/flink/runtime/rpc/resourcemanager/ResourceManager.java

```diff
@@ -138,22 +150,46 @@ public SlotAssignment requestSlot(SlotRequest slotRequest) {
 
 
 	/**
-	 *
-	 * @param resourceManagerLeaderId  The fencing token for the ResourceManager leader 
+	 * Register a taskExecutor at the resource manager
```

Comment:
Mixed camel and non-camel case. Furthermore, the `taskExecutor` is not the class name. Line break missing.